src/main/webapp/app/course/plagiarism-cases/instructor-view/detail-view/plagiarism-case-instructor-detail-view.component.ts

```diff
@@ -159,13 +173,26 @@ export class PlagiarismCaseInstructorDetailViewComponent implements OnInit, OnDe
      **/
     createEmptyPost(): void {
+        const studentName = this.plagiarismCase.student!.name;
+        const courseTitle =
+            this.plagiarismCase.exercise!.course!.title!.length > 40
+                ? this.plagiarismCase.exercise!.course!.title?.slice(0, 35) + '…'
+                : this.plagiarismCase.exercise!.course!.title;
+        const codeOfConductLink =
+            'https://www.in.tum.de/fileadmin/w00bws/in/2.Fur_Studierende/Pruefungen_und_Formalitaeten/1.Gute_studentische_Praxis/englisch/leitfaden-en_2016Jun22.pdf';
+        const apoLink =
+            'https://www.tum.de/studium/im-studium/das-studium-organisieren/satzungen-ordnungen#statute;t:Allgemeine%20Prüfungs-%20und%20Studienordnung;sort:106;page:1';
+
         this.createdPost = this.metisService.createEmptyPostForContext(undefined, undefined, undefined, this.plagiarismCase);
-        this.createdPost.content = `Dear ${this.plagiarismCase.student!.name},\nAfter a meticulous review of your final submission for the ${
-            this.plagiarismCase.exercise!.title
-        } in the course ${
-            this.plagiarismCase.exercise!.course!.title
-        }, we have concluded that you have committed plagiarism.\nThis is not only a violation of principles of good student practice but also of the “Student Code of Conduct” of the faculty of computer science that you have agreed upon. You can check the Student conduct code [here](https://www.in.tum.de/fileadmin/w00bws/in/2.Fur_Studierende/Pruefungen_und_Formalitaeten/1.Gute_studentische_Praxis/englisch/leitfaden-en_2016Jun22.pdf) §22.1 of the “Allgemeine Studien- und Prüfungsordnung (APSO)” [“General Examination and Study Regulations”] regulates consequences for such cases. You can find the APSO [here](https://www.tum.de/studium/im-studium/das-studium-organisieren/satzungen-ordnungen#statute;t:Allgemeine%20Prüfungs-%20und%20Studienordnung;sort:106;page:1).\nYou have one week to provide a statement about this situation.`;
-        this.createdPost.title = `Plagiarism Case ${this.plagiarismCase.exercise!.title}`;
+        // Note the limit of 1.000 characters for the post's content
+        this.createdPost.content =
+            `Dear ${studentName},\n\n` +
+            `After a meticulous review of your final submission for the exercise “${this.exerciseTitle}“ in the course “${courseTitle}“, we have concluded ` +
+            `that you have committed plagiarism.\nThis is not only a violation of principles of good student practice but also of the ` +
+            `[Student Code of Conduct](${codeOfConductLink}) of the faculty of computer science that you have agreed upon. ` +
+            `The *[Allgemeine Studien- und Prüfungsordnung](${apoLink})* (General Examination and Study Regulations) regulates consequences for such cases in §22.1.\n\n` +
+            `**You have one week to provide a statement about this situation.**`;
+        this.createdPost.title = `Plagiarism Case: ${this.exerciseTitle}`;
```

Comment:
I would expect this text in a markdown file and not directly in the code - what do you think?